### DISCUSSION TITLE
Regarding runtime and memory usage of solutions

### DISCUSSION THREAD

**Discussion by aryanda1** [codewars/codewars.com#2937] [General]
IS there a way to get my accepted solution runtime and memory used? How can i compare my solution with others without these?

**Comment by JohanWiltink** [comment]
There is not. To compare, you might have to reason about the respective solutions. Like, reading them, and figuring out the respective time and space complexities.

If you really can't do that without computerised analysis, copy them to your local system and do the analysis there.
